pandas/tests/test_generic.py

```diff
@@ -1365,6 +1365,23 @@ def test_spline(self):
         expected = Series([1., 2., 3., 4., 5., 6., 7.])
         assert_series_equal(result, expected)
 
+    def test_spline_extrapolate(self):
+        tm.skip_if_no_package('scipy', '0.15', 'setting ext on scipy.interpolate.UnivariateSpline')
+        s = Series([1, 2, 3, 4, np.nan, 6, np.nan])
```

Comment:
I actually can't get that method to run without the exception I link to in that comment. There seems to be no current test coverage of this method.